setup.py

```diff
@@ -26,6 +27,24 @@
 """
 
 
+class PostInstallCommand(install):
```

Comment:
I think for now this is ok, but we may want to be able to call this at the creation of an archive, and have this files controlled by Qiita rather than in the installation directory.

Reply:
We also need to think about the difference of installing this plugin and creation of an archive. These are not always identical events. Think e.g. about re-installing the plugin in Qiita when updating some internal coding. We don't want to loose the stored placements at this point.